### ISSUE TITLE
关于demo运行结果没有检测框的问题

### ISSUE THREAD

**Issue by rw1995** [DetectionTeamUCAS/FPN_Tensorflow#43]
楼主，您好，我在运行您所编辑的inference时候出现了问题
我的操作如下
1.下载resnet50_v1 到pretrained_weights文件夹下
2.在home目录下下载了voc2007的数据集（传统voc数据）
3.compile,执行了您在cython_utils文件夹下的的python setup.py build_ext --inplace命令
4.之后，我cfgs_res50_fpn.py的内容复制到cfgs.py中
5.下载权重，放置在output的trained_weights文件夹下
6.运行Demo 
python inference.py --data_dir='/home/rw/FPN_Tensorflow-master/tools/demos' --save_dir='/home/rw/FPN_Tensorflow-master/tools/inference_results/1' --GPU='0'

结果如下面所示

var_in_graph:  resnet_v1_50/block3/unit_3/bottleneck_v1/conv2/BatchNorm/moving_mean:0
var_in_ckpt:  resnet_v1_50/block3/unit_3/bottleneck_v1/conv2/BatchNorm/moving_mean
____________________________________________________________
********************************************************************************
restore from pretrained_weighs in IMAGE_NET
2019-01-22 13:24:02.512718: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-01-22 13:24:02.824773: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:897] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-01-22 13:24:02.825682: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: GeForce GTX 1060 major: 6 minor: 1 memoryClockRate(GHz): 1.733
pciBusID: 0000:01:00.0
totalMemory: 5.93GiB freeMemory: 5.53GiB
2019-01-22 13:24:02.825710: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-01-22 13:24:08.674217: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-01-22 13:24:08.674294: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-01-22 13:24:08.674315: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-01-22 13:24:08.681519: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 5294 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1060, pci bus id: 0000:01:00.0, compute capability: 6.1)
restore model
/home/rw/FPN_Tensorflow-master/tools/demos/000144.jpg image cost 9.31491208076477s:[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>]100%	1/1(tf1.10)

**我查看我所在的result/1这个文件夹中的图，发现只是图片大小发生了改变，然而并没有产生gif动图中的检测框，肯定是我的操作问题，请问下楼主。上面我那个地方没有做对？**
这没有得到我想要的检测框，希望您能帮助下我

**Comment by yangxue0827** [comment]
检查一下存的路径原来有没有，没有的话需要自己新建一下 @rw1995

**Comment by rw1995** [comment]
> 
> 
> 检查一下存的路径原来有没有，没有的话需要自己新建一下 @rw1995

我查看了，是建立的了，请问您下readme中的权重是否是放置在trained-weights文件夹中？

**Comment by yangxue0827** [comment]
你没有正确加载权重，现在你加载的是预训练权重，应该在train weights 下面还有一个文件夹。 @rw1995

**Comment by rw1995** [comment]
哇，多谢您，我已找到原因，确实是我少弄了一个文件夹
操作方法：在权重（非预训练）链接下载后，统一解压在一个文件夹下如“FPN_res50_20181201”,将这个文件夹放置下trained_weights文件夹下

**Comment by rw1995** [comment]
> 你没有正确加载权重，现在你加载的是预训练权重，应该在train weights 下面还有一个文件夹。 @rw1995

哇，多谢您，我已找到原因，确实是我少弄了一个文件夹
操作方法：在权重（非预训练）链接下载后，统一解压在一个文件夹下如“FPN_res50_20181201”,将这个**文件夹**放置下**trained_weights文件夹**下

**Comment by x2ss** [comment]
> > 你没有正确加载权重，现在你加载的是预训练权重，应该在train weights 下面还有一个文件夹。 @rw1995
> 
> 哇，多谢您，我已找到原因，确实是我少弄了一个文件夹
> 操作方法：在权重（非预训练）链接下载后，统一解压在一个文件夹下如“FPN_res50_20181201”,将这个**文件夹**放置下**trained_weights文件夹**下

Thanks for your question, and the answer given by yangxue0827. Thank yangxue0827 again for uploading this amazing work.

**Comment by x2ss** [comment]
> > 你没有正确加载权重，现在你加载的是预训练权重，应该在train weights 下面还有一个文件夹。 @rw1995
> 
> 哇，多谢您，我已找到原因，确实是我少弄了一个文件夹
> 操作方法：在权重（非预训练）链接下载后，统一解压在一个文件夹下如“FPN_res50_20181201”,将这个**文件夹**放置下**trained_weights文件夹**下

Also thanks for the operating procedure you give!
